welcome.md

```diff
@@ -0,0 +1,109 @@
+---

```

Comment:
@mhairimca it looks like you added a duplicate welcome page (see below for all of the content you added). Can you update (all items highlighted in green is what is added)

Reply:
now deleted